modmail/__init__.py

```diff
@@ -31,21 +37,22 @@
 
 file_handler.setFormatter(
     logging.Formatter(
-        fmt=fmt,
-        datefmt=datefmt,
+        fmt=FMT,
+        datefmt=DATEFMT,
     )
 )
 
 file_handler.setLevel(logging.TRACE)
 
-coloredlogs.install(
-    level=LOG_LEVEL,
-    fmt=fmt,
-    datefmt=datefmt,
-)
+# configure trace color
+LEVEL_STYLES = dict(coloredlogs.DEFAULT_LEVEL_STYLES)
```

Comment:
Could we also make these configurable, environment variables?

Reply:
I suppose that could be possible but it is an irritatingly detailed dict structure and I didn't think it was worth it.

Reply:
If we don't set this variable, coloredlogs actually will use a environment variable as override. So we could check if that env var is set, and if it is, do *not* set our level styles.

Reply:
Set up, install the optionals `environs` to use this feature